Dockerfile

```diff
@@ -15,14 +22,12 @@ COPY main.go main.go
 COPY apis/ apis/
 COPY go.mod .
 
-RUN export GOOS=$(echo ${TARGETPLATFORM} | cut -d / -f1) && \
-    export GOARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2) && \
+RUN export GOOS=$TARGETOS && \
```

Comment:
Are we double-specifying the target OS and target arch if they are also part of the `TARGETPLATFORM` variable?

Should we instead create a new variable for GOARM and get rid of `TARGETPLATFORM` altogether?

Reply:
I think we'll still need `TARGETPLATFORM` for `--platform` for builder. 
`TARGETOS`, `TARGETARCH`, etc are automatically set variables so we don't have to parse `TARGETPLATFORM`.
http://sujayopillai.com/2019/01/02/docker-platform-args.html

`TARGETVARIANT` has the `v` character in it, so we'll need to parse that if that were to be passed as `GOARM`.

Reply:
Interesting, thanks for the clarification!